src/java/io/bazel/rulesscala/scalac/ScalacWorker.java

```diff
@@ -95,6 +98,15 @@ private static Path ensureEmptyWorkDirectory(Path output, String label) throws I
 
     return Files.createDirectories(dir);
   }
+  
+  private static void ensureEmptySemanticDBDirectory(Path output, String label) throws IOException {
```

Comment:
Would it be possible to refactor `ensureEmptyWorkDirectory` instead of adding this method wich looks almost identical and serves same purpose?

Reply:
done.

(But the tests seem flaky... they have failed in different places (unrelated to this change) on last 2 runs).